packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -16,6 +16,43 @@ import '../cache.dart';
 import '../globals.dart';
 import 'sdk.dart';
 
+/// Represents Flutter-specific data that is added to the `PUB_ENVIRONMENT`
+/// environment variable and allows understanding the type of requests made to
+/// the package site on Flutter's behalf.
+// DO NOT update without contacting kevmoo.
+// We have server-side tooling that assumes the values are consistent.
+class PubContext {
+  static final RegExp _validContext = new RegExp('[a-z][a-z_]*[a-z]');
+
+  static final PubContext create = new PubContext._(<String>['create']);
+  static final PubContext createPackage = new PubContext._(<String>['create_pkg']);
+  static final PubContext createPlugin = new PubContext._(<String>['create_plugin']);
+  static final PubContext get = new PubContext._(<String>['get']);
```

Comment:
https://www.dartlang.org/guides/language/language-tour#keywords

We should use some name other than `get`